src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,19 @@
+const ADD_CATEGORY = 'bookstore/category/ADD_CATEGORY';
+const underConstruction = 'Under Construction';
+
+const initialState = [];
+
+export const addCategory = () => ({
+  type: ADD_CATEGORY,
+  text: underConstruction,
+});
+
+const categoryReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case ADD_CATEGORY:
+      return action.text;
+    default: return state;
+  }
+};
+
+export default categoryReducer;
```

Comment:
- [ ] Please as stated in the requirement, kindly use Redux Toolkit to create a slice for managing the state of `categories` in your Redux store. This [tutorial might be of help](https://redux-toolkit.js.org/tutorials/quick-start#create-a-redux-state-slice). 

<img width="651" alt="Untitled" src="https://github.com/Kwamay/React-Books/assets/103386183/9f2decdf-dfc6-4df3-992d-ce9f3a31050a">